ega-data-api-dataedge/src/test/java/eu/elixir/ega/ebi/dataedge/service/internal/RemoteFileMetaServiceImplTest.java

```diff
@@ -93,11 +95,24 @@ public void testLogDownload() {
         when(forEntityDataset.getBody()).thenReturn(datasets);
         when(forEntity.getBody()).thenReturn(files);
 
-        final File fileOutput = remoteFileMetaServiceImpl.getFile(auth, FILEID);
+        final File fileOutput = remoteFileMetaServiceImpl.getFile(auth, FILEID, "");
 
         assertThat(fileOutput.getDatasetId(), equalTo(DATASET1));
     }
 
+    @Test(expected = MethodNotAllowedException.class)
+    public void getFile_WhenFileDOesNotExist_ThenThrowsMethodNotAllowedException() {

```

Comment:
`whenFileDoesNotExist` should it be throwing a `NotFoundException` ? Is it a duplicity? 

`NotFound | 404 | The resource requested was not found`